### DISCUSSION TITLE
I'd like to know how to medddle with the new default `X` clear button ComboboxInput

### DISCUSSION THREAD

**Discussion by fellipessanha** [tailwindlabs/headlessui#2703] [Help]
I've been working on a project for some months now, and since we updated the library to use headlessui 1.7.15, the `X` button appeared in our combobox, and I coundn't figure out how to disable it.

![image](https://github.com/tailwindlabs/headlessui/assets/69586375/17dcfbca-0373-45fe-bf82-46cf27e88bde)

I'd like it disabled since, as the picture shows, we have implemented our own clear button, which has extra functionalities in our software. For instance, we have an elastic search that is triggered by onChange, but not when the button is triggered.

Could anyone give me a hint as to how I could disable it, or maybe embed it in our system? thanks :)

**Comment by ericbf** [comment]
Are you using type="search" on your input?

**Comment by barry166** [comment]
That `X` is probably not Headless UI at all. It is the browser-native clear control for an `<input type="search">`.

Use a text input for the combobox and keep your own clear button:

```tsx
<ComboboxInput
  type="text"
  value={query}
  onChange={(event) => setQuery(event.target.value)}
/>
```

If you must keep `type="search"`, hide the native control with CSS, but switching to `type="text"` is the cleaner fix because Headless UI does not need the search input type for `ComboboxInput` to work.
